terraform/ec2/localstack/main.tf

```diff
@@ -36,18 +36,18 @@ resource "aws_instance" "integration-test" {
     inline = [
       "cloud-init status --wait",
       "clone the agent and start the localstack",
-      "git clone ${var.github_repo}",
-      "cd amazon-cloudwatch-agent",
-      "git reset --hard ${var.github_sha}",
+      "git clone ${var.github_test_repo}",
+      "cd amazon-cloudwatch-agent-test",
+      "git reset --hard ${var.cwa_test_github_sha}",
       "echo set up ssl pem for localstack, then start localstack",
-      "cd ~/amazon-cloudwatch-agent/integration/localstack/ls_tmp",
+      "cd ~/amazon-cloudwatch-agent-test/localstack/ls_tmp",
       "openssl req -new -x509 -newkey rsa:2048 -sha256 -nodes -out snakeoil.pem -keyout snakeoil.key -config snakeoil.conf",
       "cat snakeoil.key snakeoil.pem > server.test.pem",
       "cat snakeoil.key > server.test.pem.key",
       "cat snakeoil.pem > server.test.pem.crt",
-      "cd ~/amazon-cloudwatch-agent/integration/localstack",
+      "cd ~/amazon-cloudwatch-agent-test/localstack",
       "docker-compose up -d --force-recreate",
-      "aws s3 cp ls_tmp s3://${var.s3_bucket}/integration-test/ls_tmp/${var.github_sha} --recursive"
+      "aws s3 cp ls_tmp s3://${var.s3_bucket}/integration-test/ls_tmp/${var.cwa_github_sha} --recursive"
```

Comment:
So how this all worked in the main repo is that part of the workflow builds the binaries and executables and uploads them to S3. This is a prerequisite step to executing the integration tests. How are we planning on keeping integration test executions up to date here when the main agent code lives in the other repo? This terraform is very likely to change since we're ripping it out from where the rest of the agent code is.

Reply:
Not following. main repo github actions steps will check out this repo and use these terraform files. Every time either of them changes if changes are incompatible main repo's github workflow should fail. Am I missing something?

Reply:
Okay then I misunderstood how this was going to be used. Makes sense. Disregard.